axelor-account/src/main/resources/views/FixedAsset.xml

```diff
@@ -434,6 +434,8 @@
     <field name="firstServiceDate" expr="eval: __config__.date"/>
     <field name="firstDepreciationDate" expr="eval: __config__.date"/>
     <field name="originSelect" expr="eval:'1'"/>
+    <field name="disposalTypeSelect" expr="eval: 0"/>
+    <field name="disposalQtySelect" expr="eval: 0"/>
```

Comment:
You don't need to add this, integer's default value is already 0
But we need an SQL script to change the default value in the database

Reply:
Fixed